utilities/test_suite/HIP/Tensor_hip.cpp

```diff
@@ -1689,5 +1712,7 @@ int main(int argc, char **argv)
     CHECK_RETURN_STATUS(hipFree(d_output));
     if(testCase == 5)
         CHECK_RETURN_STATUS(hipFree(d_interDstPtr));
+    if(testCase == 11)
```

Comment:
Instead of checking for testcase == 1
we can check if(alpha != NULL)

Reply:
Done